plugins/che-dummy-plugin/0.0.1/meta.yaml

```diff
@@ -6,3 +6,6 @@ title: Che Samples Hello World Plugin
 description: A hello world theia plug-in wrapped into a Che Plug-in
 icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
 url: https://github.com/ws-skeleton/che-dummy-plugin/releases/download/untagged-8f3e198285a2f3b6b2db/che-dummy-plugin.tar.gz
+publisher: Red Hat, Inc.
+category: Other
```

Comment:
what is the default category if there is one ? 
Could we assume that if no category is set, it goes to Other ?

Reply:
ok